script/app-mlperf-inference-intel/meta.yaml

```diff
@@ -699,7 +699,7 @@ variations:
       - tags: get,mlcommons,inference,src
         names:
         - inference-src
-      - tags: get,mlcommons,inference,loadgen,_custom-python,_keep-build
+      - tags: get,mlcommons,inference,loadgen,_custom-python,_keep-build,_inference
```

Comment:
`inference` variation is a slightly confusing name. Shall we use `_wg-inference` to make it clear?